ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayAnnotation/README.md

```diff
@@ -0,0 +1,36 @@
+# Display Annotation
+
+Display annotation from a feature service URL.
+
+![](screenshot.png)
+
+## Use case
+Annotation is useful for displaying text that you don't want to move or resize when the map is panned or zoomed (unlike labels which will move and resize). You can use annotation to place text at a fixed size, position, orientation, font, and so on. You may choose to do this for cartographic reasons or because the exact placement of the text is important.
+
+## How to use the sample
+Pan and zoom to see names of waters and burns in a small region of Scotland.
+
+## How it works
+1. Create a `ServiceFeatureTable` with a URL.
```

Comment:
These steps need to be updated once the QML code is changed. I think the same steps as C++ can be used. The only change is the wording - we don't usually say "constructed" in QML since there is no constructors. We just use the word "created" instead https://github.com/Esri/arcgis-runtime-samples-qt/pull/967/files#diff-a9e41a9d9c6a1f87da22c465352ae223R12